.travis.yml

```diff
@@ -51,7 +51,9 @@ jobs:
     - stage: Lint check
       python: "3.7"
       before_install: # Nothing to do
-      install: pip install flake8 "black==19.10b0" "isort==4.3.21" "mypy==0.782"
+      install:
+        - python setup.py install
```

Comment:
I wonder why we have to install the package at Lint stage ?

Reply:
without it `mypy` can't find any type hints, otherwise I would need to add the `ignore_missing_imports` flag for the `torch` package or add `# type: ignore` whenever torch is imported

Reply:
Maybe, let's move mypy check to https://github.com/pytorch/ignite/blob/4408a32fb676adac619dd515c440038bc4554d13/.travis.yml#L40-L41
such that we have correct environment and deps installed and Lint stage remains quick.

Reply:
ok, no problem